Directory.Build.props

```diff
@@ -1,38 +1,22 @@
 <Project>
 
-    <PropertyGroup Label="Target Platforms" >
-        <TargetFramework>net6.0</TargetFramework>
-    </PropertyGroup>
+  <PropertyGroup Label="NuGet Properties">
+    <Authors>Allegro</Authors>
+    <Company>Allegro</Company>
+    <Copyright>© Allegro. All rights reserved.</Copyright>
+    <Description>$(MSBuildProjectName)</Description>
+    <Tags>Allegro</Tags>
+    <PackageLicenseFile>LICENSE</PackageLicenseFile>
+    <PackageReadmeFile>README.md</PackageReadmeFile>
+    <PublishRepositoryUrl>true</PublishRepositoryUrl>
+    <UseArtifactsOutput>true</UseArtifactsOutput>
+  </PropertyGroup>
 
-    <PropertyGroup Label="Project Settings" >
```

Comment:
why not needed? moved somewhere?

Reply:
What I removed were mostly defaults, those which weren't were moved to the actual project file, as appropriate.